### PR TITLE
Add French translation

### PR DESCRIPTION
**Pull request by Cimbali**
Here’s a first iteration at translating to French. A couple of notes:
- I haven’t picked a name for the app yet. Keeping « Safe Paths » would possibly go over well in France, and « Chemins Sûrs » certainly better in Canada. It just sounds a bit weird to me so I’m holding off until I find a better translation.
- I noted some inconsistencies:
    - sometimes the app is called « Safe Paths » and sometimes « COVID Safe Paths ». (This also affects the previous point, I might translate it more as COVID-free paths if it is the latter.) See #484 
    - sometimes the device is called device, sometimes phone. I went with device consistently.
- I believe for proper i18n the news sources etc. should be changed to locale-specific sources.
	- I don’t know if the news feed will be localized? If so what’s the correct URL?
	- I don’t think the Mayo Clinic is known at all in France, so finding another authority as a reference would be a good idea. Institut Pasteur maybe?
	- same thing for the EULA, what’s the correct localized URL?
- “localization” could be either « géolocalisation » or « position ». I went with the latter as it is somewhat lighter and also used by Google in its French interface for data exports, but there is a case to be made for the other translation.
- translation is gender-neutral to the best of my ability
- COVID-19 is a foreign-imported neologism so it doesn’t really have a grammatical gender. I went with feminine consistently − though not the dominant use in France, it is the one recommended by the WHO and used by the Canadian government (rationale: D stands for disease and maladie is feminine).

### CODE REVIEW

**Comment by alpita-masurkar** [comment]
Hi @Cimbali, you're right, the name Safe Paths shouldn't be changed into its local translation. It's the name of the product, and easier to identify with.
Also, you can safely remove the URL keys... I think that there are 3 of them. They will fall back to the default in the original English version file.
Regarding the other questions... let me look into them. Either I or someone else will respond to those.

File: app/locales/fr.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    "event_history_title": "Historique d’exposition",
+    "export_para_1": "Si vous êtes diagnostiqué·e positif·ve à la COVID-19, partager votre historique de positions avec les autorités locales contribuera à endiguer la maladie.",
+    "export_para_2": "L’historique partagé est une simple liste de lieux et de dates, sans informations supplémentaires.",
+    "home_at_risk_header": "Vous avez pu être exposé·e",
+    "home_at_risk_subsubtext": "Ceci ne signifie pas nécessairement que vous êtes contaminés.",
+    "home_at_risk_subtext": "D’après votre historique de positions GPS, il est possible que vous ayez été en contact ou à proximité d’une personne porteuse du COVID-19.",
+    "home_enable_location": "Activer les données de position",
+    "home_mayo_link_heading": "Plus d’informations sur la COVID-19",
+    "home_mayo_link_label": "de l’ONG Mayo Clinic",
+    "home_mayo_link_URL": "https://www.mayoclinic.org/coronavirus-covid-19",
```
Comment (alpita-masurkar) [inline app/locales/fr.json:30]: can remove this key and the other 2 URL keys

**Comment by alpita-masurkar** [comment]
@heri you had done the last French translation. can you review these?

**Comment by tstirrat** [comment]
Let me know when this is ready to merge

File: app/locales/fr.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    "latest_news": "Dernières nouvelles",
+    "launch_done_header": "Terminé",
+    "launch_done_subheader": "Tout est prêt. Rappelez vous que vous pouvez toujours revenir ajuster vos préférences.",
+    "launch_enable_location": "Activer la position",
+    "launch_enable_notif": "Activer les notifications",
+    "launch_finish_set_up": "Terminer la configuration",
+    "launch_get_started": "Commencer",
+    "launch_location_access": "Accès à la position",
+    "launch_location_header": "Pour se rappeler où vous êtes allé·e, votre appareil doit enregistrer vos positions",
+    "launch_location_subheader": "Ne vous inquiétez pas, cette aucune information ne quitte votre appareil sans que vous la partagiez explicitement.",
```
Comment (heri) [inline app/locales/fr.json:46]: Ne vous inquiétez pas, aucune information ne quitte votre appareil sans votre partage explicite.

File: app/locales/fr.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    "launch_screen1_header": "Le retour à la normal commence ici.",
+    "launch_screen2_header": "Recevez une notification si vous avez croisé une personne qui a été ultérieurement diagnostiquée positive à la COVID-19.",
+    "launch_screen2_subheader": "Le savoir, c'est le pouvoir.",
+    "launch_screen3_header": "Si vous êtes diagnostiqué·e positif·ve à la COVID-19, vous pourrez partager vos données anonymement",
+    "launch_screen3_subheader": "ce qui contribuera à endiguer la maladie autour de vous.",
+    "launch_screen4_header": "Vous êtes aux commandes. Vos données ne sont sauvegardées que sur votre appareil.",
+    "launch_screen4_subheader": "Si vous êtes diagnostiqué·e positif·ve, vous seul·e pourrez choisir de partager vos données.",
+    "launch_set_up_phone": "Configurer mon appareil",
+    "legal_page_title": "Juridique",
+    "less_than_one_minute": "moins d’1 minute",
```
Comment (heri) [inline app/locales/fr.json:60]: moins d'une minute

File: app/locales/fr.json [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    "authorities_no_sources": "Aucune source ajoutée",
+    "authorities_removal_alert_cancel": "Annuler",
+    "authorities_removal_alert_desc": "Êtes-vous sûr·e de vouloir retirer cette autorité de sources de données?",
+    "authorities_removal_alert_proceed": "Continuer",
+    "authorities_removal_alert_title": "Supprimer une autorité",
+    "authorities_title": "Sources de confiance",
+    "choose_provider_subtitle": "Pour vous tenir informé·e des risques d’exposition, vous devez souscrire à une autorité sanitaire.",
+    "choose_provider_title": "Choisissez votre autorité sanitaire",
+    "commitment": "Engagement",
+    "commitment_para": "Safe Paths enregistre et vérifie de façon sécurisée vos interactions avec autrui en utilisant vos données de position. Vos données ne quitteront JAMAIS votre appareil sans votre consentement.",
```
Comment (heri) [inline app/locales/fr.json:17]: "donnée de position" doesn't sound French native. 

I would use "coordonnées" which is used often.

File: app/locales/fr.json [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    "default_news_site_name": "Nouvelles SafePaths (en anglais)",
+    "event_history_subtitle": "Comprendre son exposition personnelle à partie des informations partagées par les autorités sanitaires.",
+    "event_history_title": "Historique d’exposition",
+    "export_para_1": "Si vous êtes diagnostiqué·e positif·ve à la COVID-19, partager votre historique de positions avec les autorités locales contribuera à endiguer la maladie.",
+    "export_para_2": "L’historique partagé est une simple liste de lieux et de dates, sans informations supplémentaires.",
+    "home_at_risk_header": "Vous avez pu être exposé·e",
+    "home_at_risk_subsubtext": "Ceci ne signifie pas nécessairement que vous êtes contaminés.",
+    "home_at_risk_subtext": "D’après votre historique de positions GPS, il est possible que vous ayez été en contact ou à proximité d’une personne porteuse du COVID-19.",
```
Comment (heri) [inline app/locales/fr.json:25]: positions GPS => coordonnées GPS

File: app/locales/fr.json [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    "home_enable_location": "Activer les données de position",
+    "home_mayo_link_heading": "Plus d’informations sur la COVID-19",
+    "home_mayo_link_label": "de l’ONG Mayo Clinic (en anglais)",
+    "home_next_steps": "En savoir plus",
+    "home_no_contact_header": "Pas de contacts connus",
+    "home_no_contact_subtext": "D’après les données disponibles, vous n’avez pas été à proximité de porteur·ses du COVID-19.",
+    "home_unknown_header": "Pas de données",
+    "home_unknown_subtext": "Il est nécessaire d’autoriser l’application à accéder à votre position pour déterminer les risques d’exposition.",
+    "import_step_1": "1. Connectez vous à votre compte Google et téléchargez votre Historique des positions",
```
Comment (heri) [inline app/locales/fr.json:34]: Connectez-vous

File: app/locales/fr.json [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+    "nCoV2019_url_info": "Plus d’informations sur les données représentées sur cette carte",
+    "news_subtitle": "Obtenez les dernières nouvelles sur le COVID-19 par vos autorités sanitaires et en général.",
+    "news_title": "Dernières nouvelles",
+    "no_data": "Pas de données",
+    "notification_2_weeks_ago": "Il y a 2 semaines",
+    "notification_data_not_available": "Pas de données d’exposition disponibles.",
+    "notification_random_data_button": "Choisir une autorité sanitaire",
+    "notification_title": "Profil des expositions sur 2 semaines",
+    "notification_today": "aujourd’hui",
+    "notification_warning_text": "Si une autorité sanitaire dans votre région a mis en place Safe Paths vous pouvez abonner pour obtenir des mises à jours régulières sur les risques d’exposition.",
```
Comment (heri) [inline app/locales/fr.json:79]: Si une autorité sanitaire dans votre région a mis en place Safe Paths, vous pouvez vous abonner pour obtenir des mises à jours régulières sur les risques d’exposition.

File: app/locales/fr.json [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+    "notifications_exposure_format": "Vous avez croisé une personne infectée il y a {{daysAgo}} jours pendant {{exposureTime}} minutes.",
+    "notifications_exposure_format_today": "Vous avez croisé une personne infectée aujourd’hui pendant {{exposureTime}} minutes.",
+    "notifications_exposure_format_yesterday": "Vous avez croisé une personne infectée hier pendant {{exposureTime}} minutes.",
+    "notifications_no_exposure": "Pas d’expositions connues à la COVID-19 durant les deux dernières semaines.",
+    "overlap_found_button_label": "Données publiques chargées",
+    "overlap_no_results_button_label": "Données publiques chargées",
+    "overlap_para_1": "Le chemin vert représente votre historique de positions\n\nLes cercles violets représentes les données publiques",
```
Comment (heri) [inline app/locales/fr.json:86]: Les cercles violets pâles

File: app/locales/fr.json [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+    "overlap_title": "Chercher les recoupements",
+    "push_at_risk_message": "Vous avez croisé une personne infectée à la COVID-19",
+    "push_at_risk_title": "Vous avez pu être exposé",
+    "settings_title": "Réglages",
+    "share_location_data": "Partager les données de position",
+    "show_overlap": "Cliquez pour voir les données publiques",
+    "team": "L’équipe",
+    "team_para": "Notre équipe est un consortium composé d’épidémiologistes, d’ingénieur·es, de scientifiques des données, de défenseurs de la vie privée numérique, de professeur·es et de chercheur·ses d’institutions réputées: dont: Le MIT, Harvard, The Mayo Clinic, TripleBlind, EyeNetra, Ernst & Young, et Link Ventures.",
```
Comment (heri) [inline app/locales/fr.json:94]: institutions réputées, dont le MIT, Harvard, The Mayo Clinic, TripleBlind, EyeNetra, Ernst & Young et Link Ventures.

**Comment by tstirrat** [comment]
Thanks!

@alpita-masurkar or @Cimbali can you edit this PR with changes?

File: app/locales/fr.json (hunk 4 above)
Comment (Cimbali) [reply app/locales/fr.json]: Position is almost definitely fine in native French. The sentence structure does make it sound weird, so I changed that.

Plus it’s also coherent with the google export that is suggested
image

File: app/locales/fr.json (hunk 5 above)
Comment (Cimbali) [reply app/locales/fr.json:25]: Same as above

**Pull request merged**